static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -171,27 +171,24 @@ define([
 
     function ContributionsABTestVariant(options, test) {
         var trackingCampaignId = test.epic ? 'epic_' + test.campaignId : test.campaignId;
+        var campaignCode = getCampaignCode(test.campaignPrefix, test.campaignId, this.id, test.campaignSuffix);
 
-        this.campaignId = test.campaignId;
         this.id = options.id;
-        this.maxViews = options.maxViews || maxViews;
-        this.isUnlimited = options.isUnlimited || false;
-
-        this.pageviewId = (config.ophan && config.ophan.pageViewId) || 'not_found';
-        this.campaignCode = getCampaignCode(test.campaignPrefix, this.campaignId, this.id, test.campaignSuffix);
-        this.campaignCodes = [this.campaignCode];
-
-        this.contributeURL = options.contributeURL || this.getURL(contributionsBaseURL, this.campaignCode);
-        this.membershipURL = options.membershipURL || this.getURL(membershipBaseURL, this.campaignCode);
-
-        this.componentName = 'mem_acquisition_' + trackingCampaignId + '_' + this.id;
-
-        this.template = options.template || controlTemplate;
-
-        this.blockEngagementBanner = options.blockEngagementBanner || false;
-        this.engagementBannerParams = options.engagementBannerParams || {};
-
-        this.isOutbrainCompliant = options.isOutbrainCompliant || false;
+        this.options = {
```

Comment:
there are a few places outside of this file that access these properties, so will need updating to use `options`